src/jdk.jcmd/share/classes/sun/tools/jstat/OptionFormat.java

```diff
@@ -77,15 +77,14 @@ public String getName() {
 
     public void apply(Closure c) throws MonitorException {
 
-      for (Iterator<OptionFormat> i = children.iterator(); i.hasNext(); /* empty */) {
-          OptionFormat o = i.next();
-          c.visit(o, i.hasNext());
-      }
+        for (Iterator<OptionFormat> i = children.iterator(); i.hasNext(); /* empty */) {
+            OptionFormat o = i.next();
```

Comment:
Why did not you simplify the lines 80-81 the same way as in line 85?

Reply:
It can't be simplified: it calls `Iterator.hasNext()` inside cycle body.

Reply:
Okay. Thank you for explanation.